extension/sanitizer/sanitizer/Sanitizers/Floats.zep

```diff
@@ -30,8 +30,7 @@ class Floats extends Sanitizer implements SanitizerInterface, SizeInterface {
 
         let this->val = round(floatval(v), this->maxDecimals);
 
-
-        if (this->val === 0.00) {
+        if intval(this->val) === 0 {
```

Comment:
`(int) this->val` is faster than `intval(this->val)`

Reply:
:+1: